test/check-application

```diff
@@ -2065,6 +2069,213 @@ class TestFiles(testlib.MachineCase):
         b.wait_in_text("h4.pf-v5-c-alert__title", "Pasting failed")
         b.wait_in_text(".pf-v5-c-alert__description", "\"newfile\" exists")
         self.assertEqual(m.execute("head -n 1 /home/admin/kbdCopy/newfile"), "keybindings good\n")
+        b.click(".pf-v5-c-alert__action button")
+
+        # Copy & paste as superuser
+        # Switch to list so owner is visible
+        b.click("button[aria-label='Display as a list']")
+        b.go("/files#/?path=/home/admin")
+        self.assert_last_breadcrumb("admin")
+        m.execute("""
+            useradd -m foouser
+            runuser -u admin echo 'test copy' > /home/admin/newfile
+        """)
+
+        b.click("[data-item='newfile']")
+        b.click("#dropdown-menu")
+        b.click("#copy-item")
+
+        b.go("files#/?path=/home/foouser")
+        self.assert_last_breadcrumb("foouser")
+        # Paste file as destination directory owner
+        b.click("#dropdown-menu")
+        b.click("#paste-item")
+        b.wait_visible("#paste-owner-modal")
+        b.click("#paste-owner-modal button.pf-m-primary")
+        b.wait_in_text("[data-item='newfile'] .item-owner", "foouser")
+        self.assertEqual(m.execute("head -n 1 /home/foouser/newfile"), "test copy\n")
+
+        # Copy foouser file into admin directory as admin
+        m.execute("runuser -u foouser -- bash -c 'echo footext > /home/foouser/foouserfile'")
+        b.wait_in_text("[data-item='foouserfile'] .item-owner", "foouser")
+        b.click("[data-item='foouserfile']")
+        b.click("#dropdown-menu")
+        b.click("#copy-item")
+        b.go("files#/?path=/home/admin")
+        self.assert_last_breadcrumb("admin")
+        b.click("#dropdown-menu")
+        b.click("#paste-item")
+        b.wait_visible("#paste-owner-modal")
+        b.click("#paste-owner-modal button.pf-m-primary")
+        b.wait_in_text("[data-item='foouserfile'] .item-owner", "admin")
+        self.assertEqual(m.execute("head -n 1 /home/admin/foouserfile"), "footext\n")
+
+        # Paste as admin into foouser directory
```

Comment:
This just swaps "admin" and "foouser", is that a new scenario somehow? Feels like the same as the previous one.

Reply:
The previous scenario is that user doesn't have access to source path, here it's the destination path.
It is also different from the one before scenario as it pastes with different ownership.